app/uk/gov/hmrc/gform/validation/ComponentValidator.scala

```diff
@@ -554,6 +554,23 @@ object ComponentValidator {
     else validationSuccess
   }
 
+  def validateChoiceNoneError[D <: DataOrigin](
+    fieldValue: FormComponent,
+    noneChoice: Int,
+    error: LocalisedString
+  )(
+    formModelVisibilityOptics: FormModelVisibilityOptics[D]
+  )(implicit l: LangADT): ValidatedType[Unit] = {
+    val choiceValue = formModelVisibilityOptics.data
+      .get(fieldValue.modelComponentId)
```

Comment:
Please call `many` not `get`. 

`many` is for components which can produce multiple answers, which in our case is only choice

Reply:
ok, I got inspiration from 
https://github.com/hmrc/gform-frontend/blob/e7d469c09104c79f3039b75270ff3affe5fe6809/app/uk/gov/hmrc/gform/validation/ComponentValidator.scala#L548
should we update it here as well?

Reply:
Yes, we should change it there as well.